stubs/cachetools/cachetools/func.pyi

```diff
@@ -9,5 +9,5 @@ def lru_cache(maxsize: float = ..., typed: bool = ...) -> IdentityFunction: ...
 def mru_cache(maxsize: float = ..., typed: bool = ...) -> IdentityFunction: ...
 def rr_cache(maxsize: float = ..., choice: Callable[[Sequence[_T]], _T] | None = ..., typed: bool = ...) -> IdentityFunction: ...
```

Comment:
All `foo_cache` functions accept `None`.

Reply:
And apparently they accept some kind of function too, because the source code does `if callable(maxsize)`, but that can be added in a separate PR.

Reply:
I forgot about that haha, was only looking at the function I needed. I will add the other None type hints now.

For the callable part, I wasn't sure what kind of callable this function needed - like what kind of arguments and return type does this callable contain etc. If you know it, do let me know and I can add it in this PR as well.